weather_sp/splitter_pipeline/file_splitters.py

```diff
@@ -147,37 +148,35 @@ def split_data(self) -> None:
                                   for var in dataset.data_vars])
             else:
                 iterlists.append([dataset])
-            for dim in ('time', 'level'):
-                if dim in self.output_info.split_dims():
-                    iterlists.append(dataset[dim])
+            filtered_split_dims = [
+                x for x in self.output_info.split_dims() if x not in ('variable', self._UNSUPPORTED_DIMENSIONS)]
+            for dim in filtered_split_dims:
+                iterlists.append(dataset[dim])
```

Comment:
I think it would be clearer to have this be a single for loop with an if statement. Something like: 
```suggestion
            filtered_split_dims = [
            for dim in self.output_info.split_dims():
                if dim not in ('variable', self._UNSUPPORTED_DIMENSIONS):
                    iterlists.append(dataset[dim])
```

Reply:
I reused the filtered_split_dims to create the output file though. So I'd have to do similar filtering (or at least check for 'variable' at that point again.